src/components/AssociatedTagVariantsTable.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { OtTable, commaSeparate } from 'ot-ui';
 
-const tableColumns = [
+const tableColumns = variantId => [
   {
     id: 'tagVariantId',
     label: 'Variant',
     renderCell: rowData => (
       <Link to={`/variant/${rowData.tagVariantId}`}>
-        {rowData.tagVariantId}
+        {variantId === rowData.tagVariantId
```

Comment:
Here it would be better just not to render the `Link` component if `variantId === rowData.tagVariantId`.